common/content/options.js

```diff
@@ -41,7 +41,7 @@ const Option = Class("Option", {
 
         // add no{option} variant of boolean {option} to this.names
         if (this.type == "boolean")
-            this.names = array([name, "no" + name] for (name in values(names))).flatten().__proto__;
+            this.names = util.Array.flatten(Array([name, "no" + name] for (name in values(names)))).__proto__;
```

Comment:
Ah, ok. So, the reason why you can't access it is because `const` and `let` was changed. Previously, it was basically like `var` but now it makes the variable only available in the file.
See https://github.com/vimperator/vimperator-labs/pull/387
Would you want to update this PR along those lines? Thanks.